components/dss-tools/plugins/org.wso2.integrationstudio.artifact.dataservice/src/org/wso2/integrationstudio/artifact/dataservice/ui/wizard/GenerateDataServicesUtils.java

```diff
@@ -0,0 +1,270 @@
+    }
+
+    /**
+     * This method creates Data Update SQL Statement
+     * <p/>
+     * UPDATE [table name] SET Select_prev = 'Y',Update_prev = 'Y' where [field
+     * name] = 'user';
+     * 
+     * @param param2
+     *            -List of parameters values
+     * @param tableName
+     *            -table Name of the given table
+     * @param schema the database schema
+     * @param pKey
+     *            -primary key of table
+     * @return SQL statement as a String
+     */
+    public static String getUpdateStatement(String tableName, String schema, List<String> param2,
+            List<String> pKeys) {
+        List<String> param = new ArrayList<String>();
+        for (String par : param2) {
+            if (!pKeys.contains(par))
+                param.add(par);
+        }
+
+        StringBuffer statement = new StringBuffer();
+        statement.append("UPDATE ");
+        statement.append((isEmptyString(schema) ? "" : (schema + ".")) + tableName.trim());
+        statement.append(" SET ");
+        int last = param.size();
+        int index = 1;
+        for (String par : param) {
+
+            statement.append(par).append("=?");
+            if (index != last)
+                statement.append(",");
+
+            index++;
+        }
+        statement.append(" WHERE ");
+        int i = 0;
+        for (String pKey : pKeys) {
+            if (i > 0) {
+                statement.append(" AND ");
+            }
+            statement.append(pKey).append("=?");
+            i++;
+        }
+        return new String(statement);
+    }
+
+    /**
+     * This method creates Data Delete SQL Statement
+     * <p/>
+     * DELETE from [table name] where [field name] = 'whatever';
+     * 
+     * @param tableName
+     *            -table Name of the given table
+     * @param schema the database schema
+     * @param pKey
+     *            -primary key of table
+     * @return SQL statement as a String
+     */
+    public static String getDeleteStatement(String tableName, String schema, List<String> pKeys) {
+        StringBuffer statement = new StringBuffer();
+        statement.append("DELETE FROM ");
+        statement.append((isEmptyString(schema) ? "" : (schema + ".")) + tableName.trim());
+        statement.append(" WHERE ");
+        int i = 0;
+        for (String pKey : pKeys) {
+            if (i > 0) {
+                statement.append(" AND ");
+            }
+            statement.append(pKey).append("=?");
+            i++;
+        }
+        return new String(statement);
+    }
+    
+    /**
+     * Check the given text is empty or not.
+     *
+     * @param text The text to be checked
+     * @return true if text is null or trimmed text length is empty, or else false
+     */
+    private static boolean isEmptyString(String text) {
+        if (text != null && text.trim().length() > 0) {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
Shall we simplify this as `return text == null || text.trim().length() == 0;` ?

Reply:
Done https://github.com/wso2/integration-studio/pull/890/commits/7c67b03c80855728fbecedbfaad2e6f19af7f2c6